12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,35 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+	[[:lower:]]*
 
   2. All files whose name starts with an uppercase letter
   Answer:
+  [[:upper:]]*
 
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
+  *[aeiou]
 
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
+  *[!aeiou]
 
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
+  src/*.c
 
   6. All files with extension ".c" under any folder
   Answer:
+  */*.c
 
 2. Write a command that would delete all files ending with a ".png" under any subdirectory.
+find . -type f -iname *.png -delete
 
 3. The command "ls -l" can take the name of a file or directory and show details on them. From within "wildcards" folder run "ls -l [A-Z]*". What was printed? Why?
+What is "wildcards" folder?
+Basically the command should show details of all files starting with a any letter.
```

Comment:
It should only list files starting with an **uppercase letter**, but it might list all except for ```a``` [I try to explain here](url).